packages/backend/src/models/DashboardModel/DashboardModel.ts

```diff
@@ -876,4 +879,137 @@ export class DashboardModel {
             .distinctOn(`${DashboardVersionsTableName}.dashboard_id`)
             .where(`${ProjectTableName}.project_uuid`, projectUuid);
     }
+
+    private async checkDashboardTileExistsInDashboard(
```

Comment:
I perform this check when getting comments and creating a comment, lmk if it's overkill @rephus 👍

Reply:
this is done below